src/providers/pass-certificate-validator/pass-certificate-validator.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class PassCertificateValidatorProvider {
+
+  isPassCertificateValid(certificate: string | null): boolean {
+    if (!certificate || certificate.length !== 8) {
```

Comment:
This number 8 appears in multiple files. Could we have a constant variable for it somewhere and just reference that instead?

Reply:
created and reference a constant file.